docs/extend/authorization.md

```diff
@@ -249,4 +251,5 @@ Response body           | []byte            | Raw docker daemon response body
 Name    | Type   | Description
 --------|--------|----------------------------------------------------------------------------------
 Allow   | bool   | Boolean value indicating whether the response is allowed or denied
-Message | string | Authorization message (will be returned to the client in case the access is denied)
\ No newline at end of file
+Message | string | Authorization message (will be returned to the client in case the access is denied)
+Error   | string | Error message (will be returned to the client in case the plugin encounter an error)
```

Comment:
@thaJeztah take a look at the `Name` column above as well, you'll notice they're not the Go/JSON fields, but just names see https://github.com/docker/docker/pull/18662/files#diff-2ff682c22254f5a72f6f87eae3080154R239